app/controllers/api/account.php

```diff
@@ -1069,6 +1068,7 @@
             /* {{user}} ,{{team}} and {{project}} are required in the templates */
```

Comment:
lets add it back to the comment. and mention not to remove it

Reply:
on it